.github/workflows/changelog_to_tag.yml

```diff
@@ -74,3 +75,8 @@ jobs:
           body_path: ./.tagmsg.txt
           draft: false
           prerelease: false
+      - name: Publish role to Galaxy
```

Comment:
The mssql role doesn't currently have this task to publish the role to Galaxy - is that because we don't publish a legacy role, but only publish a collection?

Reply:
Yes exactly